include/zephyr/lorawan/lorawan.h

```diff
@@ -337,6 +362,20 @@ void lorawan_get_payload_sizes(uint8_t *max_next_payload_size,
  */
 int lorawan_set_region(enum lorawan_region region);
 
+/**
+ * @brief Configure channels used in the network
+ *
+ * Configure the channels that are used with the gateway(s) that are used in
+ * the network. Since not all gateways neccesarily used all 64 channels, this
+ * should be set to prevent sending uplinks over channels that are not used.
+ * This should be called before @a lorawan_join().
```

Comment:
"before @ a lorawan_join()" typo?

Reply:
The `@a` creates a link in doxygen syntax.